types/conductor-animate/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for conductor-animate 1.0
+// Project: https://github.com/TaeKimJR/conductor-animate#readme
+// Definitions by: Tae Kim <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+import * as React from 'react';
+
+interface Animations {
+    [K: string]: React.ComponentClass<any> | React.FunctionComponent<any>;
+}
+
+interface Config {
+    animation: string;
```

Comment:
Should we add the following properties?

```
delay?: number;
duration: number;
```

Although not strictly required, I think most consumers will want to use this information. Your example animations use these properties.